src/Transform/ONNX/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ target_include_directories(OMElideConstants
         PRIVATE ${ONNX_MLIR_SRC_ROOT} ${ONNX_MLIR_BIN_ROOT}
         ${ONNF_MLIR_SRC_ROOT})
 
+add_dependencies(OMElideConstants
+        OMONNXOpsIncGen ShapeInferenceOpInterfaceIncGen)
```

Comment:
I think we should say OMElideConstants depends on ONNXOps. The dependencies specified here are transitive dependencies, and we do not want to manage them by ourselves (imagine how many places you need to modify in the build script if ONNXOps takes on an additional dependency).